pkg/cmd/run.go

```diff
@@ -88,6 +97,29 @@ getenvoy run standard:1.11.1 -- --help
 			}
 
 			key, manifestErr := manifest.NewKey(args[0])
+
+			// Check if the templateArgs were passed to the cmd line.
+			// If they were passed, config must be created based on
+			// template.
+			if len(templateParams) > 0 {
+				// When template params are specified, config should not be in envoy params
+				for _, envoyParam := range args {
+					if strings.HasPrefix(envoyParam, "--config") {
+						return fmt.Errorf("--templateArg and %s cannot be specified at the same time", envoyParam)
+					}
+				}
+				config, err := flavors.CreateConfig(key.Flavor, templateParams)
```

Comment:
I did a typo running getenvoy and triggered a panic here (used `postgres/nightly` instead of `postgres:nightly`)

```
 ./getenvoy run postgres/nightly --templateArg Endpoint=127.0.0.1 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xe55895]

goroutine 1 [running]:
github.com/tetratelabs/getenvoy/pkg/cmd.NewRunCmd.func2(0xc000417b80, 0xc0004343c0, 0x1, 0x3, 0x0, 0x0)
        /home/lizan/github/tetratelabs/getenvoy/review/pkg/cmd/run.go:111 +0x205
github.com/spf13/cobra.(*Command).execute(0xc000417b80, 0xc000434360, 0x3, 0x3, 0xc000417b80, 0xc000434360)
        /home/lizan/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:826 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x19b42c0, 0xc00009e778, 0xc00035ff78, 0x405abf)
        /home/lizan/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /home/lizan/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
        /home/lizan/github/tetratelabs/getenvoy/review/cmd/getenvoy/main.go:26 +0x2b
```

Reply:
I will look at this.

Reply:
Fixed.